api/app/Console/Commands/CreateDigitalCareersPools.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\User;
+use App\Models\Pool;
+use App\Models\Skill;
+use App\Models\Classification;
+use Database\Helpers\ApiEnums;
+use Illuminate\Console\Command;
+use Carbon\Carbon;
+
+class CreateDigitalCareersPools extends Command
+{
+    // this command creates 36 pools, 4 classifications by 9 streams
```

Comment:
Actually 48 now!  4x8

Reply:
huh

Reply:
Lol, I'm a dummy.  My db client returned 48 rows (36 + the original 12).  Time to take a break...  :coffee: